### ISSUE TITLE
Remove image/inacessible

### ISSUE THREAD

**Issue by ChristopherChudzicki** [mitodl/ocw-studio#1157]
Remove all instances of
```
![This resource may not render correctly in a screen reader.](/images/inacessible.gif)
```
as described in comment
> FYI, I just met with the OL accessibility expert and she told me that we should not continue using the hidden image with alt tag to flag "inaccessible" PDFs. She claims that many more PDFs are accessible than we think, and that it's no safe for us to assume what a user can or wants to get out of file. It's sufficient to indicate that it's a PDF. And, if we really wanted to be helpful, we would have a note that explains the content of the PDF, rather then just tell users "this isn't for you."

https://github.com/mitodl/ocw-studio/issues/1130#issuecomment-1072704670

These links:
- show missing image icon in studio
- are bad for accessibility
- are noise for link fixing / broken link detection
- are extremely low effort to remove, so lets do it.
